QtAwesome/iconic_font.py

```diff
@@ -1,22 +1,81 @@
 """Classes handling iconic fonts"""
 
 from __future__ import print_function
-from PyQt4.QtCore import Qt, QObject, QPoint, QRect, qRound, QByteArray
+from PyQt4.QtCore import Qt, QObject, QPoint, QRect, qRound, QByteArray, QTimer
 from PyQt4.QtGui import (QIcon, QColor, QIconEngine, QPainter, QPixmap,
                          QFontDatabase, QFont)
 import json
 import os
 
+
 _default_options = {
     'color': QColor(50, 50, 50),
     'scale_factor': 0.9,
 }
 
 
-class CharIconPainter:
+class CharIconAnimator:
```

Comment:
Maybe, but so far I do not see any gain... right now I am storing the info of the animations inside the `CharIconPainter` in `self.animator = CharIconAnimator()`, using 2 classes for that seems overkill..., I would still need to know which class to initialize, so maybe I can change and if for a dictionary, but there will be some choosing...